roslaunch_editor/README.md

```diff
@@ -0,0 +1,28 @@
+# roslaunch_editor
+
+Web-based ROS launch-files editor, created for making configuration of your robot more user-friendly for novices.
+
+<img src="roslaunch_editor.jpg" width=450>
+
+## Quick launch
+
+```bash
+roslaunch roslaunch_editor example.launch
+```
+
+Then, open `http://localhost:8085/roslaunch_editor/` and edit the test launch file.
+
+## Modes
+
+`roslaunch_editor` works in two modes: standalone mode (where running `editor` node is required), and Clover mode (where `clover`'s `shell` node, `roswww_static` and `rosbridge_suite` are utilized). The editor will read and write launch-files and restart the nodes (if configured) using one of these nodes.
```

Comment:
Honestly, I'd prefer a single (standalone) mode. We've discussed `shell` before, and even though I understand its usefulness, I'm not in favor of it being used.